docker/pymisp2/Dockerfile

```
@@ -5,4 +5,6 @@ COPY requirements.txt .
 
 RUN pip install --upgrade pip \
  && apk --update add --no-cache --virtual .build-dependencies python3-dev gcc linux-headers libc-dev libffi-dev\
- build-base wget git && pip install --no-cache-dir -r requirements.txt && apk del .build-dependencies
+ build-base wget git && pip install --no-cache-dir -r requirements.txt && apk del .build-dependencies && pip uninstall -y extract-msg\
+ && pip uninstall -y pcodedmp\
+ && find . -type f -name 'emailobject.py' -exec sed -i -e '/from extract_msg import openMsg/d; /from extract_msg.message import Message as MsgObj/d; s/msg_obj: MsgObj/msg_obj/g' {} \;
```

Using `find ... exec` is not a good choice. As if it doesn't find the file it doesn't fail. You actually don't need `find`. You can simply use a glob. Use the following:

 ```suggestion
 && sed -i -e '/from extract_msg import openMsg/d; /from extract_msg.message import Message as MsgObj/d; s/msg_obj: MsgObj/msg_obj/g' /usr/local/lib/python3*/site-packages/pymisp/tools/emailobject.py
```

This will also fail if no file is found (for example if pymisp change emailobject.py file's name).

---

Done